app/components/settings/categories/general-setting/ThemeSettingsBlock.js

```diff
@@ -112,26 +112,24 @@ export default class ThemeSettingsBlock extends Component<Props> {
         <p><FormattedMessage {...messages.blog} values={{ blogLink }} /></p>
 
         <div className={styles.main}>
+          {/* @Todo: Theme Preview Enumeration should be more dynamic? */}
```

Comment:
What do you mean by more dynamic?

Reply:
As in automatically iterate all the themes that exist instead of the current code which hard-codes the existence of two theme. Not really worth spending time on I think

Reply:
It was added by original author, deleted.